### ISSUE TITLE
npx nestjs-admin createAdminUser throwing an error

### ISSUE THREAD

**Issue by BorisDutkin** [Theodo-UK/nestjs-admin#163]
running the command throws exception:

```
Nest cannot export a provider/module that is not a part of the currently processed module (AdminUserModule). Please verify whether the exported AdminAuthModuleFactory is available in this particular context.

Possible Solutions:
- Is AdminAuthModuleFactory part of the relevant providers/imports within AdminUserModule?
 +21ms
Error: Nest cannot export a provider/module that is not a part of the currently processed module (AdminUserModule). Please verify whether the exported AdminAuthModuleFactory is available in this particular context.

Possible Solutions:
- Is AdminAuthModuleFactory part of the relevant providers/imports within AdminUserModule?

    at Module.validateExportedProvider (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\injector\module.js:275:19)      
    at addExportedUnit (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\injector\module.js:242:67)
    at Module.addExportedProvider (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\injector\module.js:253:9)
    at NestContainer.addExportedProvider (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\injector\container.js:119:19) 
    at DependenciesScanner.insertExportedProvider (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\scanner.js:199:24)   
    at C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\scanner.js:107:50
    at Array.forEach (<anonymous>)
    at DependenciesScanner.reflectExports (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\scanner.js:107:17)
    at DependenciesScanner.scanModulesForDependencies (C:\Users\boris\Documents\development\nest\omer-project\node_modules\@nestjs\core\scanner.js:59:18)    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by serkyron** [comment]
I'm having the same issue.

**Comment by valerybugakov** [comment]
same issue here after upgrading to `@nestjs/core@^6.0.0`

**Comment by digniserv** [comment]
We're having the same issue with NestJS 7.0

**Comment by Chup4Chups** [comment]
I confirm I have the same issue
 "@nestjs/core": "^7.0.0",

**Comment by hackyxicor** [comment]
facing same issue...

any lead?

**Comment by fstn** [comment]
I have the same issue here

**Comment by hackyxicor** [comment]
> facing same issue...
> 
> any lead?

fixed the issue by downgrading version

**Comment by amoussard** [comment]
Same issue here.
Someone has a workaround ? I would prefer to stay on nestjs 7.

**Comment by yuriizinets** [comment]
@amoussard 
Seems that providing custom auth is the only way to bypass issue for now. Just follow the docs: https://github.com/Theodo-UK/nestjs-admin/blob/master/docs/custom-authentication.md

**Comment by JaviOverflow** [comment]
any updates on this? I'm having the same error

**Comment by TheBigBadBunce** [comment]
Fixed in #188. Updating to version 0.4.1 should fix the problem!
